src/Behat/Symfony2Extension/Context/Argument/ServiceArgumentResolver.php

```diff
@@ -108,15 +112,18 @@ private function getServiceName($argument)
     }
 
     /**
-     * @param ContainerInterface $container
-     * @param string $argument
-     * @return string
+     * Sanitise the container key given by the Behat config file,
+     * and retrieve the parameter from the Container.
+     *
+     * @param  ContainerInterface $container
+     * @param  string $key
+     * @return mixed
      */
-    private function replaceParameters(ContainerInterface $container, $argument)
+    private function replaceParameters(ContainerInterface $container, $key)
```

Comment:
As you change the behaviour of this method, can you also change its name? Also, why this name change (`key`) ?